pkg/cmd/server/kubernetes/master.go

```diff
@@ -74,12 +75,21 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) ([]string, error
 	}
 
 	messages := []string{}
+	// v1 has to be printed separately since it's served from different endpoint than groups
 	if configapi.HasKubernetesAPIVersion(c.Options, v1.SchemeGroupVersion) {
-		messages = append(messages, fmt.Sprintf("Started Kubernetes API at %%s%s", KubeAPIPrefixV1))
+		messages = append(messages, fmt.Sprintf("Started Kubernetes API at %%s%s/%s", KubeAPIPrefix, v1.SchemeGroupVersion.String()))
```

Comment:
change this back... we don't want to assume group version's string serialization is how you build URL paths